tests/foreman/api/test_bookmarks.py

```diff
@@ -0,0 +1,290 @@
+"""Test classes for Bookmark tests"""
+# -*- encoding: utf-8 -*-
+from robottelo.constants import BOOKMARK_ENTITIES
+from robottelo.decorators import stubbed, tier1
+from robottelo.test import APITestCase
+
+
+class BookmarkTestCase(APITestCase):
+    """Test for common Bookmark operations via API"""
+
+    # CREATE TESTS
+    @stubbed()
+    @tier1
+    def test_positive_create_bookmark(self):
```

Comment:
I don't think the name of the test needs to include the name of the entity being created. It should be implicit from the module name. Instead, based on the docstring, perhaps the name should be `test_positive_create_with_name_and _contoller`?